src/views/Player.vue

```diff
@@ -60,14 +64,10 @@ export default {
         'failsafeType': 'g-form',
         'failsafeUrl': ''
       },
-      isSeekBarMoving: true,
-      played: [],
-      submitted: [],
-      revised: [],
-      skipped: [],
-      paused: [],
-      enterFullscreen: [],
-      exitFullscreen: []
+      fullscreen_journey: [],
+      question_journey: [],
+      playback_journey: [],
```

Comment:
Replaced these with journey arrays.

playback_journey -- contains info about -- played, paused, seeked
fullscreen journey -- contains info about -- enter/exit fullscreen
question journey -- contains info about -- at what time, what action was taken by the student